sound/soc/sof/intel/hda-mlink.c

```diff
@@ -773,6 +773,18 @@ int hda_bus_ml_suspend(struct hdac_bus *bus)
 }
 EXPORT_SYMBOL_NS(hda_bus_ml_suspend, SND_SOC_SOF_HDA_MLINK);
 
+struct mutex *hdac_bus_eml_get_mutex(struct hdac_bus *bus, bool alt, int elid)
+{
+	struct hdac_ext2_link *h2link;
+
+	h2link = find_ext2_link(bus, alt, elid);
+	if (!h2link)
+		return NULL;
+
+	return &h2link->eml_lock;
```

Comment:
nitpick: I would be using a positive expression ;)
```c
if (h2link)
	return &h2link->eml_lock;

return NULL;
```

But probably this aligns more with the functions where you are bailing out in case the h2link is not found.

Reply:
yes, I followed the pattern, no change here